vpn-proxy/app/urls.py

```diff
@@ -5,6 +5,10 @@
 
 urlpatterns = [
     url(r'^$', views.tunnels, name='tunnels'),
+    # /target_IP/target_port/interface_id/
+    url(r'(?P<target>([0-9]{1,3}.){3}[0-9]{1,3})/'
+        r'(?P<port>[0-9]+)/'
+        r'(?P<tunel_id>[0-9]+)/$', views.connection, name='connection'),
```

Comment:
That's not a very sane API endpoint. If anything, it should be something like `{tunnel_id}/forwardings/{target}/{port}`